llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -271,6 +272,28 @@ static void collectKernelModuleMap(
   }
 }
 
+// Collect all the dependencies for the function.
+static bool collectFunctionCallGraphNodes(llvm::Function *Func) {
```

Comment:
The name and comment do not represent the actual content: it says "collect", but return type if `bool`, the other says "call graph nodes", but it actually just detect a call to `__devicelib_assert_fail`